aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -207,13 +415,73 @@ module aptos_std::ristretto255_bulletproofs {
         );
 
         // This will fail with error::invalid_argument(E_DESERIALIZE_RANGE_PROOF)
-        verify_range_proof_pedersen(&com, proof, num_bits, A_DST);
+        assert!(verify_range_proof_pedersen(&com, proof, num_bits, A_DST) == false, 1);
     }
 
     #[test(fx = @std)]
-    fun test_valid_range_proof_verifies_against_comm(fx: signer) {
+    fun test_empty_batch_range_proof(fx: signer) {
+        features::change_feature_flags_for_testing(&fx, vector[ features::get_bulletproofs_batch_feature() ], vector[]);
+
+        let proof = &range_proof_from_bytes(vector[ ]);
+        let num_bits = 64;
+        let comms = vector[pedersen::new_commitment_for_bulletproof(
+            &ristretto255::scalar_one(),
+            &ristretto255::new_scalar_from_sha2_512(b"hello random world")
+        )];
+
+        // This will fail with error::invalid_argument(E_DESERIALIZE_RANGE_PROOF)
+        assert!(verify_batch_range_proof_pedersen(&comms, proof, num_bits, A_DST) == false, 1);
```

Comment:
Please fix this to abort with the same deserialization error code as the native for individual proof verification.